dspMetadataGUI/util/utils.py

```diff
@@ -313,13 +313,36 @@ class Datatype(Enum):
     SHORTCODE = 16
     """Shortcode"""
 
+    @classmethod
+    def is_string_like(cls, type) -> bool:
+        """Return True if type is sting or string-like"""
+        if type in [cls.STRING,
+                    cls.STRING_OR_URL,
+                    cls.URL,
+                    cls.EMAIL,
+                    cls.DOWNLOAD,
+                    cls.PLACE]:
+            return True
+        return False
+
+    @classmethod
+    def is_dropdownable(cls, type) -> bool:
+        """Return True if type is a reference to an object or controlled vocabulary (i.e. if it goes into a dropdown in the GUI)."""
+        if type in [cls.PERSON_OR_ORGANIZATION,
+                    cls.PERSON,
+                    cls.ORGANIZATION,
```

Comment:
PERSON_OR_ORGANIZATION, PERSON and ORGANIZATION seems somewhat redundant, can it be reduced to only PERSON and ORGANIZATION?